registry/registry.go

```diff
@@ -25,12 +25,15 @@ const DefaultKeyPrefix = "/_coreos.com/fleet/"
 
 // EtcdRegistry fulfils the Registry interface and uses etcd as a backend
 type EtcdRegistry struct {
-	etcd      etcd.Client
-	keyPrefix string
+	etcdClient etcd.Client
```

Comment:
It's not, that was from when I was originally moving things around, and when I thought I'd have to mess with the Registry code more. I can revert that back.